engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableWhereTest.java

```diff
@@ -269,23 +271,55 @@ public void testWhereInDependency() {
             boolean flushed = UpdateGraphProcessor.DEFAULT.flushOneNotificationForUnitTests();
             TestCase.assertTrue(flushed);
 
+            // to get table 1 satisfied we need to still fire a notification for the filter execution, then the combined
+            // execution
+            if (QueryTable.FORCE_PARALLEL_WHERE) {
+                // the merged notification for table 2 goes first
+                flushed = UpdateGraphProcessor.DEFAULT.flushOneNotificationForUnitTests();
+                TestCase.assertTrue(flushed);
+
+                System.out.println("Flushing parallel notifications for setTable1");
```

Comment:
Are you wanting to keep these `println()` calls in the log or are they debug helpers?

Reply:
I turned these into log.debug invocations. I suspect they would be helpful if somehow parallel processing gets stuck and you'd like to better understand what those threads are supposed to be doing.